src/Symfony/Component/Security/Core/Authorization/ExpressionLanguageProvider.php

```diff
@@ -14,6 +14,10 @@
 use Symfony\Component\ExpressionLanguage\ExpressionFunction;
 use Symfony\Component\ExpressionLanguage\ExpressionFunctionProviderInterface;
 
+if (!interface_exists('Symfony\Component\ExpressionLanguage\ExpressionFunctionProviderInterface')) {
```

Comment:
Done really need that logic on this class? Looks superfluous to me, but I may be missing something.

Reply:
you are right, this class also used in ExpressionLanguage.